apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V72__fiss_audit_trail_rda_position.sql

```diff
@@ -0,0 +1 @@
+alter table rda.fiss_audit_trails add rda_position smallint not null;
```

Comment:
Isn't `smallint` only 2 bytes in PSQL?  If so the field should be a short too.  But if the proto file says `uint32` this column should probably be `int` instead.

Also we might have to do a dance in the migration because this is `not null` and there will be records in the table already.  Maybe set a default on it to `0` or something and then drop the default?